hadoop-ozone/client/src/main/java/org/apache/hadoop/ozone/client/rpc/RpcClient.java

```diff
@@ -1431,6 +1412,62 @@ public OzoneOutputStream createKey(
     return createOutputStream(openKey);
   }
 
+  @Override
+  public OzoneOutputStream rewriteKey(String volumeName, String bucketName, String keyName,
+      long size, long existingKeyGeneration, ReplicationConfig replicationConfig,
+      Map<String, String> metadata) throws IOException {
+    if (keyName == null) {
+      throw new IllegalArgumentException("Key cannot be null");
+    }
```

Comment:
This is also checked in `createKeyPreChecks`.

Reply:
Ah, well spotted. I must have missed removing that when I created the precheck method!